### PR TITLE
feat(organize): allow moving/copying of multiple files/folders at the same time

### PR DESCRIPTION
**Pull request by max-wittig**
this feature is a big time-saver for organizing files

### CODE REVIEW

**Comment by max-wittig** [comment]
Could somebody review this PR?

File: src/www/ui/template/admin_content_move.html.twig [hunk 1]
```diff
@@ -22,8 +22,8 @@
       <form name="formy" method="post">
         <input type="hidden" name="fromFolder" id="fromFolder" value="-1"/>
 
-        <label for="foldercontent">{{ 'Select the upload or folder you wish to move'|trans }}:<br/>
-          <select name="foldercontent" id="foldercontent" size="20">
+        <label for="foldercontent">{{ 'Select uploads or folders you wish to move'|trans }}:<br/>
+          <select name="foldercontent[]" id="foldercontent" multiple="multiple" size="20">
```
Comment (maximilianhuber) [inline src/www/ui/template/admin_content_move.html.twig]: What do the square brackets mean?

Comment (max-wittig) [inline src/www/ui/template/admin_content_move.html.twig]: Multiselect. Otherwise it doesn't work. Try it without. 
I think it's a PHP specific thing

Comment (max-wittig) [inline src/www/ui/template/admin_content_move.html.twig]: Found a stackoverflow post, but nothing specifiy as to why it is needed.
http://stackoverflow.com/questions/30819489/select-multiple-not-working

Comment (maximilianhuber) [inline src/www/ui/template/admin_content_move.html.twig]: Yes

> They address a limitation of PHP, which doesn't generate an array automatically if multiple values with the same name are submitted, [...]

Comment (max-wittig) [inline src/www/ui/template/admin_content_move.html.twig]: so this functionality isn't working as it appears to work?
Multiple select is also possible, but there are no square brackets[]. Is it even sending all selected files?
That's another issue, but maybe a potenial bug. 
image

**Pull request merged**